include/drivers/i2c.h

```diff
@@ -553,6 +560,29 @@ static inline int z_impl_i2c_slave_driver_unregister(const struct device *dev)
 	return api->driver_unregister(dev);
 }
 
+#ifdef CONFIG_I2C_GET_CONFIG
```

Comment:
Can you move this adjacent to i2c_configure()?

Reply:
Done